test/hotspot/jtreg/vmTestbase/nsk/share/gc/gp/GarbageUtils.java

```diff
@@ -86,6 +87,22 @@ public boolean accept(String errorMessage) {
         private GarbageUtils() {
         }
 
+        public static void engageGC(ExecutionController stresser, long testMemory) {
+            // Number of Young GC iterations before triggering full GC.
+            final long YOUNG_GC_ITERATIONS = 100;
+            final long memChunk = testMemory / YOUNG_GC_ITERATIONS;
+            int iteration = 0;
+
+            while (stresser.continueExecution()) {
+                while (iteration++ < YOUNG_GC_ITERATIONS) {
+                    byteArrayProducer.create(memChunk);
```

Comment:
it is needed to create at least some old objects which live after young GC.
I propose to add 'tmp = byteArrayProducer.create(memChunk);' and free this tmp variable tight before WB.fullGC().
So we will have objects with different age (up to to 100).